File: Codesnippet/Codesnippet_php/UnitTest.php
<?php

namespace Adyen\Tests\Unit;

use Adyen\AdyenException;
use Adyen\Model\Checkout\Amount;
use Adyen\Model\Checkout\CreateCheckoutSessionRequest;
use Adyen\Service\Checkout\PaymentsApi;
use PHPUnit\Framework\TestCase;

class SnippetTest extends TestCase
{
    public function testSnippetSuccess()
    {
        // Set up the client and service.
        $client = new \Adyen\Client();

        //has issue with $Bk in API key use single quotes?
        $client->setXApiKey('AQEshmfxLI3HbhJLw0m/n3Q5qf3VeIdYCppEfGBbyawB/isehnLWag91Hdz+6UkQwV1bDb7kfNy1WIxIIkxgBw==-Kphz51iO9Nk6tKXEw5F2OYFfs0dGnzgbHLHq54YFcak=-9+yyk%f:@K$Bk=c;');
        $client->setEnvironment(\Adyen\Environment::TEST);
        $client->setApplicationName('Test Application');
        $client->setTimeout(30);

        $service = new PaymentsApi($client);
        
        // Creating Amount Object
        $amount = new Amount();
        $amount
            ->setValue(1500)
            ->setCurrency("EUR");

        // Create the actual Request
        $sessionRequest = new CreateCheckoutSessionRequest();
        $sessionRequest
            ->setMerchantAccount("PluginDemo_Djoyke_TEST")
            ->setAmount($amount)
            ->setReference("payment-test")
            ->setReturnUrl("https://your-company.com/...");
        
        $result = $service->sessions($sessionRequest);
        // Assert that the result is not null
        $this->assertNotNull($result);
    }
}

File: Codesnippet_java/unitTest.java
//import the required class
package com.adyen;

import com.adyen.Client;
import com.adyen.service.checkout.PaymentsApi;
import com.adyen.model.checkout.Amount;
import com.adyen.model.checkout.CreateCheckoutSessionRequest;
import com.adyen.model.checkout.CreateCheckoutSessionResponse;
import com.adyen.enums.Environment;
import org.junit.Test;

import static org.junit.Assert.assertNotNull;

public class SnippetTest extends BaseTest
{
    @Test
    public void SnippetTest() throws Exception
    {
        //Setup Client and Service
        Client client = new Client("YOUR_API_KEY", Environment.TEST);

        //create session request
        CreateCheckoutSessionRequest sessionRequest = new CreateCheckoutSessionRequest();
        PaymentsApi checkoutPaymentsApi = new PaymentsApi(client);
        Amount amount = new Amount().currency("EUR").value(1000L);
        sessionRequest.setAmount(amount);
        sessionRequest.setReturnUrl("https://your-company.com/checkout?shopperOrder=12xy...");
        sessionRequest.setMerchantAccount("YOUR_MERCHANT_ACCOUNT");
        sessionRequest.setReference("YOUR_PAYMENT_REFERENCE");
        sessionRequest.setCountryCode("NL");
        CreateCheckoutSessionResponse createCheckoutSessionResponse = checkoutPaymentsApi.sessions(sessionRequest);

        System.out.println("Create Checkout Session Response: " + createCheckoutSessionResponse);
        assertNotNull(createCheckoutSessionResponse);
    }
}